pkg/handlers/generic/lifecycle/csi/aws-ebs/handler.go

```diff
@@ -35,22 +35,22 @@ func (a *AWSEBSConfig) AddFlags(prefix string, flags *pflag.FlagSet) {
 	)
 }
 
-type AWSEBS struct {
+type CSI struct {
```

Comment:
We're using CSI too often as a struct type. Also, can you remove these changes and keep this scoped to CCM?